README.mdt

```diff
@@ -70,11 +69,13 @@ Note that the version number refers to the major version of the SDK you are usin
 
 The `app_origin` should be the location at which your website is located. The origin consists of the protocol and the host of your site only, without the resource path.
 
-Previously, redirect URIs had the following form:
+Before v2.2.0, redirect URIs were available at the following location:
```

Comment:
I actually liked the original wording you had for this one :grimacing: because the redirect URI isn't "available" at a location